diff --git a/pkg/desktop-appdata/speedcrunch.appdata.xml.in b/pkg/desktop-appdata/speedcrunch.appdata.xml.in
index 85918a23..0026faec 100644
--- a/pkg/desktop-appdata/speedcrunch.appdata.xml.in
+++ b/pkg/desktop-appdata/speedcrunch.appdata.xml.in
@@ -6,6 +6,11 @@
   <project_license>GPL-2.0+</project_license>
   <update_contact>speedcrunch@googlegroups.com</update_contact>
   <content_rating type="oars-1.0" />
+  <developer id="org.speedcrunch">
+    <_name>The SpeedCrunch Developers</_name>
+  </developer>
+
+  <launchable type="desktop-id">speedcrunch.desktop</launchable>
 
   <!-- TRANSLATORS: Name of the application, usually shouldn't be translated. -->
   <_name>SpeedCrunch</_name>
@@ -31,6 +36,7 @@
 
   <screenshots>
     <screenshot type="default">
+      <_caption>Main window with some example calculations</_caption>
       <image>https://heldercorreia.bitbucket.io/speedcrunch/_static/appdata-screenshots/0.12/advanced-features.png</image>
     </screenshot>
     <screenshot>
@@ -62,11 +68,7 @@
     <release version="0.10" date="2008-04-01"/>
   </releases>
 
-  <!-- We can't use the <translations> tag since our qm files are baked into the
-       executable so we just use a static list instead. See:
-       https://blogs.gnome.org/hughsie/2016/01/25/appdata-and-the-gettext-domain/#comment-5137
-       TODO: Maybe autogenerate this. -->
-  <translation/>
+  <!-- TODO: Maybe autogenerate this. -->
   <languages>
     <lang>hu</lang>
     <lang>pt_PT</lang>
diff --git a/pkg/speedcrunch.appdata.xml b/pkg/speedcrunch.appdata.xml
index f2a345c0..03297eab 100644
--- a/pkg/speedcrunch.appdata.xml
+++ b/pkg/speedcrunch.appdata.xml
@@ -5,7 +5,11 @@
   <project_license>GPL-2.0+</project_license>
   <update_contact>speedcrunch@googlegroups.com</update_contact>
   <content_rating type="oars-1.0"/>
+  <developer id="org.speedcrunch">
+    <name>The SpeedCrunch Developers</name>
+  </developer>
 
+  <launchable type="desktop-id">speedcrunch.desktop</launchable>
   
   <name>SpeedCrunch</name>
   <name xml:lang="de">SpeedCrunch</name>
@@ -60,6 +64,7 @@
 
   <screenshots>
     <screenshot type="default">
+      <caption>Main window with some example calculations</caption>
       <image>https://heldercorreia.bitbucket.io/speedcrunch/_static/appdata-screenshots/0.12/advanced-features.png</image>
     </screenshot>
     <screenshot>
@@ -112,7 +117,6 @@
   </releases>
 
   
-  <translation/>
   <languages>
     <lang>hu</lang>
     <lang>pt_PT</lang>
@@ -150,4 +154,4 @@
     <lang>vi</lang>
     <lang>en</lang>
   </languages>
-</component>
\ No newline at end of file
+</component>
